components/cycles/AddWorkouts/AddWorkouts.jsx

```diff
@@ -0,0 +1,230 @@
+      ],
+    },
+    {
+      name: 'Arms',
+      lastPerformed: 7,
+      id: 4,
+      muscleGroups: 'Biceps Triceps',
+      color: '#9D8DFF',
+      exercises: [
+        { id: 'BICEP CURLS', sets: [{ weight: '25', reps: '20' }, { weight: '25', reps: '20' }, { weight: '25', reps: '20' }, { weight: '25', reps: '20' }] },
+        { id: 'TRICEP DIPS', sets: [{ weight: '25', reps: '20' }, { weight: '25', reps: '20' }, { weight: '25', reps: '20' }, { weight: '25', reps: '20' }] },
+        { id: 'HAMMER CURLS', sets: [{ weight: '25', reps: '20' }, { weight: '25', reps: '20' }, { weight: '25', reps: '20' }, { weight: '25', reps: '20' }] },
+        { id: 'SKULL CRUSHERS', sets: [{ weight: '25', reps: '20' }, { weight: '25', reps: '20' }, { weight: '25', reps: '20' }, { weight: '25', reps: '20' }] },
+      ],
+    },
+  ];
+
+  const parsedItems = parseItems(items);
+  const [workoutCount, setWorkoutCount] = React.useState(0);
+  const [workoutsList, setWorkoutsList] = React.useState(parsedItems);
+  const [addedWorkouts] = React.useState([]);
+
+  const renderCard = ({ item: { left, right }, index }) => (
+    <WorkoutCardPair>
+      <SelectableWorkoutCard
+        name={left.name}
+        subtext={left.subtext}
+        onPress={() => {
+          const temp = { ...workoutsList };
+          const { selected } = temp[left.name[0].toUpperCase()][index].left;
+          temp[left.name[0].toUpperCase()][index].left.selected = !selected;
+          setWorkoutsList(temp);
+
+          (temp[left.name[0].toUpperCase()][index].left.selected === true) ? setWorkoutCount(workoutCount + 1)
+            : setWorkoutCount(workoutCount - 1);
+          (temp[left.name[0].toUpperCase()][index].left.selected === true) ? addedWorkouts.push(left)
+            : addedWorkouts.splice(addedWorkouts.indexOf(left), 1);
+        }}
+        selected={left.selected}
+        displayAddButton={left.displayAddButton}
+        color={left.color}
+        key={left.name + left.subtext}
+      />
+
+      {right ? (
+        <SelectableWorkoutCard
+          name={right.name}
+          subtext={right.subtext}
+          selected={right.selected}
+          onPress={() => {
+            const temp = { ...workoutsList };
+            const { selected } = temp[right.name[0].toUpperCase()][index].right;
+            temp[right.name[0].toUpperCase()][index].right.selected = !selected;
+            setWorkoutsList(temp);
+            (temp[right.name[0].toUpperCase()][index].right.selected === true) ? setWorkoutCount(workoutCount + 1)
+              : setWorkoutCount(workoutCount - 1);
+            (temp[right.name[0].toUpperCase()][index].right.selected === true) ? addedWorkouts.push(right)
+              : addedWorkouts.splice(addedWorkouts.indexOf(right), 1);
+          }}
+          displayAddButton={right.displayAddButton}
+          color={right.color}
+          key={right.name + right.subtext}
+        />
+      ) : (
```

Comment:
Might want to make a more empty one instead of using the same component, bc this component has a box shadow which I think would still show up.

Reply:
I just checked and it doesn't show the box shadow, you're right tho I feel like it should but it doesn't so we're good here hahah